src/components/AboutUsIntroduction/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import useTranslation from 'next-translate/useTranslation'
+
+const AboutUsIntroduction = () => {
+  const { t } = useTranslation('about-us')
+  return (
+    <div className="about-us-introduction-section">
+      <h2 className="about-us-introduction-section__title-intro">
+        {t('aboutUsSectionTitleIntro')}
+      </h2>
+      <h2 className="about-us-introduction-section__title-description">
+        {t('aboutUsSectionTitleDescription')}
+      </h2>
+      <div className="about-us-introduction-section__decoration-box-long"></div>
+      <div className="about-us-introduction-section__image"></div>
+      <div className="about-us-introduction-section__decoration-box-short"></div>
+      <p className="about-us-introduction-section__description">
+        {t('aboutUsSectionDescription')}
+      </p>
+    </div>
+  )
```

Comment:
Using empty `div` to render image is also nice, but the trade of is there are too many empty `div` for the browser to render when you can do it like this.
```suggestion
  const AboutUsIntroduction = () => {
  const { t } = useTranslation('about-us')
  const Component = (props) => <h1 className="about-us-introduction__title" {...props} />

  return (
    <section className="about-us-section">
      <div className="about-us-introduction about-us-introduction--background">
        <Trans
          i18nKey={t('aboutUsSectionTitle')}
          components={[<Component />, <br/>]}
        />
        <p className="about-us-introduction__description">
          {t('aboutUsSectionDescription')}
        </p>
      </div>
    </section>
  )
}
```

Reply:
Firstly, `about-us` is separated into **sections** so `introduction` is also a `section`.

Reply:
Next, from my comment in `en/about-us.json`, you will understand how `<Trans>` work.

Reply:
Moreover, no more empty `div`s when all can be added as a background image. See this:

<img width="937" alt="Screen Shot 2564-02-24 at 17 06 57" src="https://user-images.githubusercontent.com/32289827/108984740-d9846e80-76c2-11eb-9b2d-c8a4247c956e.png">

The **handshake** image is a `background` for `about-us-introduction` while for the description us can use `padding` to squeeze the text and apply a `background-color`.

<img width="510" alt="Screen Shot 2564-02-24 at 17 07 06" src="https://user-images.githubusercontent.com/32289827/108984993-1badb000-76c3-11eb-80b3-6c1e70183a06.png">